images/operator/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.14.7 as builder
+FROM golang:1.15 as builder
```

Comment:
refrain from making changes to the go version now

Reply:
We should probably discuss this tomorrow.

Reply:
Done!